fe/fe-core/src/main/java/com/starrocks/meta/SqlBlackList.java

```diff
@@ -35,8 +35,9 @@ public static SqlBlackList getInstance() {
     }
 
     public static void verifying(String sql) throws AnalysisException {
+        String formatSql = sql.replace("\r", " ").replace("\n", " ").replaceAll("\\s+", " ");
         for (BlackListSql patternAndId : getInstance().sqlBlackListMap.values()) {
-            Matcher m = patternAndId.pattern.matcher(sql);
+            Matcher m = patternAndId.pattern.matcher(formatSql);
```

Comment:
We only remove space when building balcklist pattern. Should we also need to remove line sperator here?
```
public void analyze() throws SemanticException {
        sql = sql.trim().toLowerCase().replaceAll(" +", " ");
        if (sql.length() > 0) {
            try {
                sqlPattern = Pattern.compile(sql);
            } catch (PatternSyntaxException e) {
                throw new SemanticException("Sql syntax error: %s", e.getMessage());
            }
        }
    }
```

Reply:
We need stricter syntax rule checking when the user adds the sql to the Backlist, so I don't think we need to remove the separator here.

Reply:
We try not to modify user inputs in case they are confused

Reply:
Yes, Added